src/modules/comments.test.js

```diff
@@ -0,0 +1,17 @@
+import { commentsCounter } from './counter.js';
+
+jest.mock('./requests');
+
+describe('commens counter test', () => {
+  test('should render comments counter ', () => {
+    document.body.innerHTML = '<p>Comments (<span class="comments"></span>)</p>';
+
+    commentsCounter([
+      { creation_date: '2022-11-24', username: 'jo', comment: 'bo' },
+      { creation_date: '2022-11-24', username: 'mo', comment: 'bo' },
+    ]);
+
+    const commentsText = document.querySelector('.comments').textContent;
+    expect(Number(commentsText)).toEqual(2);
```

Comment:
I think is a good idea to test edge cases. One of them could be to test the counter when you have 0 comments. :+1: :smiley: :smiley: